container/charliecloud/parpe_base/install_parpe.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -e 
+cd
+git clone --single-branch --branch feature_charlie --depth=1 https://github.com/ICB-DCM/parPE.git
+cd parPE
+export PARPE_BASE=`pwd`
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use $(..) instead of legacy `..`.](https://app.codacy.com/app/dweindl/parPE/pullRequest?prid=3661955)